sql/sqlfmt.go

```diff
@@ -0,0 +1,111 @@
+// Copyright 2023 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sql
+
+import (
+	"fmt"
+	"strings"
+)
+
+func GenerateCreateTableStatement(tblName string, colStmts []string, tblCharsetName, tblCollName string) string {
```

Comment:
Give these public/exported funcs some short function docs and this'll be good to go. As a general rule, I think any public/exported function is worth documenting. 

I think the function docs can be very short and sweet... think about what will be useful to a new teammate trying to use these functions... for example... GenerateCreateTableStatements could mention that `colStmts` is intended to be the results of calling `GenerateCreateTableColumnDefinition`. Because we have several functions involved in building a create table statement correctly, I think it's worth giving future readers some hints on how to use these functions together, so they don't have to go and try to reverse engineer the code and figure it out.